### DISCUSSION TITLE
Welcome to Pixel-Playgroud-Repo Discussions!

### DISCUSSION THREAD

**Discussion by AtlasAlexander** [AlexAlmuaini/Pixel-Playgroud-Repo#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Saito197** [comment]
Hi, I found this while browsing "issue" and "good first issue", can you tell me what this project is about and if you need/want any outsider contribution?

**Reply by AtlasAlexander** [reply]
Hi, thanks for being interested, I didn't realise the repository was set to public. This repo is for a university project so we are unable to accept external support. But again thank you & I'll let you know if I have any personal projects that I need help with :D
